backend/src/services/stripe/stripe.service.ts

```diff
@@ -0,0 +1,67 @@
+import StripeApi from 'stripe';
+
+import {
+  PaymentCurrency,
+  PaymentMethod,
+  PaymentMode,
+} from '~/common/enums/enums';
+
+const QUANTITY_OF_ITEMS = 1;
+const OPERATION_NAME = 'Replenish your funds on GuruHub';
+const CENTS_IN_ONE_DOLLAR = 100;
```

Comment:
How about moving it to separate enum, like PaymentUnitAmount? In that case it'll be easier to introduce other currencies in future

Reply:
I already have an enum for currencies

Reply:
Yes, but I'm talking about the units, in this case CENTS_IN_ONE_DOLLAR